docs/porting/libraries.md

```diff
@@ -1,5 +1,304 @@
-# 🔧 Porting your libraries
+# Porting Libraries to .NET Core
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+By [Phillip Carter](https://github.com/cartermp)
+
+Libraries are some of the most natural projects to port from .NET Framework to .NET Core.  By not necessarily being tied to a particular framework or application model, libraries can often be simpler to port to .NET Core.  This article covers all of the steps you'll need to take to get to the point where you can focus on your code.  It covers the following:
```

Comment:
Nit:
I'll just say it once and you can apply to all your docs.
When used as a noun (with or without the version number), always precede .NET Framework with the definite article the (the .NET Framework…).
Also, remove extra space from before This article.

This sentence "This article covers all of the steps you'll need to take to get to the point where you can focus on your code." also bugs me a bit. I have to think of a suggestion.